### PR TITLE
constexpr conversions

### PR DESCRIPTION
**Pull request by neheb**
Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: src/web/config_load.cc [hunk 1]
```diff
@@ -296,10 +296,10 @@ void web::configLoad::process()
         pr++;
     }
 
-    std::vector<config_option_t> autoscanList = {
+    constexpr config_option_t autoscanList[] = {
```
Comment (whyman) [inline src/web/config_load.cc:299]: Could we use std::array here too?

Comment (neheb) [reply src/web/config_load.cc:299]: Part of the problem is that some of the members are ifdef'd out. std::array requires a size.

Comment (neheb) [reply src/web/config_load.cc]: Converted to std::array

**Comment by whyman** [comment]
Thanks!

**Pull request merged**